src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_Image.cs

```diff
@@ -515,6 +520,55 @@ public async Task When_ImageFailed()
 			await WindowHelper.WaitFor(() => imageFailedRaised);
 		}
 
+#if __SKIA__
+		[TestMethod]
+		[RunsOnUIThread]
+		public async Task When_Png_Should_Have_High_Quality()
+		{
+			var image = new Image() { Width = 100, Height = 100 };
+			await UITestHelper.Load(image);
+			bool imageOpenedRaised = false;
+			image.ImageOpened += (s, e) =>
+			{
+				imageOpenedRaised = true;
+			};
+
+			image.Source = new BitmapImage(new Uri("ms-appx:///Assets/Icons/star_empty.png"));
+
+			await WindowHelper.WaitFor(() => imageOpenedRaised);
+
+			var storageFile = await StorageFile.GetFileFromApplicationUriAsync(new Uri("ms-appx:///Assets/Icons/star_empty.png"));
```

Comment:
@Youssef1313 tests are only done with **.png** here, do we need to test more cases? (png with transparency, without transparency, jpg, etc...)

Reply:
png is enough. It's the same way to load any image